doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1728,11 +1728,11 @@ Declaring Implicit Arguments
    To know which are the implicit arguments of an object, use the
    command :cmd:`Print Implicit` (see :ref:`displaying-implicit-args`).
 
-.. warn:: Argument number @num is a trailing implicit so must be maximal.
+.. exn:: Argument id is a trailing implicit, so it can't be declared non maximal. Please use %{ %} instead of [ ].
```

Comment:
Shouldn't `id` be `@ident` or something like that? @coq/doc-maintainers will know better.

Reply:
Seems to work indeed.

Reply:
Actually currently we do not parse such notations in warning / error messages. But feel free to still use them. The pattern is so common that we should absolutely parse those.

Reply:
Actually, I was wrong. We do parse these notations. It is just the warning / error index that does not display them correctly, which we should fix.